spec/memo_wise_spec.rb

```diff
@@ -400,6 +412,13 @@ def public_memowise_method
       }
     end
 
+    context "when the method to preset memoization for is not memoized" do
+      it {
```

Comment:
I have a slight preference for these multiline blocks to be `do...end`s instead of `{...}`. How would you feel about changing them (here and below)?

Reply:
Same preference here! I'm surprised that rubocop-rspec doesn't lint that?

Reply:
Also, are these `it` blocks not named because RSpec magic creates a name from the expectation?